handlers/telesom/telesom.go

```diff
@@ -69,17 +69,22 @@ func (h *handler) receiveMessage(ctx context.Context, channel courier.Channel, w
 func (h *handler) SendMsg(ctx context.Context, msg courier.Msg) (courier.MsgStatus, error) {
 	username := msg.Channel().StringConfigForKey(courier.ConfigUsername, "")
 	if username == "" {
-		return nil, fmt.Errorf("no username set for Telesom channel")
+		return nil, fmt.Errorf("no username set for TS channel")
 	}
 
 	password := msg.Channel().StringConfigForKey(courier.ConfigPassword, "")
 	if password == "" {
-		return nil, fmt.Errorf("no password set for Telesom channel")
+		return nil, fmt.Errorf("no password set for TS channel")
 	}
 
 	privateKey := msg.Channel().StringConfigForKey(courier.ConfigSecret, "")
 	if privateKey == "" {
-		return nil, fmt.Errorf("no private key set for Telesom channel")
+		return nil, fmt.Errorf("no private key set for TS channel")
+	}
+
+	tsSendURL := msg.Channel().StringConfigForKey(courier.ConfigSendURL, "")
```

Comment:
can just be: 
  `msg.Channel().StringConfigForKey(courier.ConfigSendURL, sendURL)` no?

Reply:
I was thinking that would miss if "" is set in the config but we can make sure the URL is required on the claim form